netsync/manager.go

```diff
@@ -1207,55 +1181,55 @@ out:
 				}
 
 			case isCurrentMsg:
-				msg.reply <- b.current()
+				msg.reply <- sm.current()
 
 			case pauseMsg:
 				// Wait until the sender unpauses the manager.
 				<-msg.unpause
 
 			default:
-				bmgrLog.Warnf("Invalid message type in block "+
+				log.Warnf("Invalid message type in block "+
 					"handler: %T", msg)
 			}
 
-		case <-b.quit:
+		case <-sm.quit:
 			break out
 		}
 	}
 
-	b.wg.Done()
-	bmgrLog.Trace("Block handler done")
+	sm.wg.Done()
+	log.Trace("Block handler done")
 }
 
 // handleBlockchainNotification handles notifications from blockchain.  It does
 // things such as request orphan block parents and relay accepted blocks to
 // connected peers.
-func (b *blockManager) handleBlockchainNotification(notification *blockchain.Notification) {
+func (sm *SyncManager) handleBlockchainNotification(notification *blockchain.Notification) {
```

Comment:
This is definitely not something that needs to be addressed in this PR, but I wonder if this wouldn't make more sense to have in the mempool package now with it having a way to query if the sync manager believes it is current.  I believe that is really the only reason it requires the sync manager.  Everything else is related to updating the mempool and relay.

Reply:
Yeah, I'll keep that in mind as I refactor this package.